confluent_kafka/src/Consumer.c

```diff
@@ -391,12 +391,18 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 
 static PyObject *Consumer_close (Handle *self, PyObject *ignore) {
 	CallState cs;
+	int raise = 0;
 
 	CallState_begin(self, &cs);
 
 	rd_kafka_consumer_close(self->rk);
 
-	if (!CallState_end(self, &cs))
+	raise = !CallState_end(self, &cs);
```

Comment:
@edenhill Hmm, I guess it doesn't matter much then. Seems fine as is and this LGTM.